org.buhocms.BuhoCMS.yml

```diff
@@ -0,0 +1,51 @@
+app-id: org.buhocms.BuhoCMS
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: buhocms
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+modules:
+  - name: BuhoCMS
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/BuhoCMS
+      - cp -r buhocms data lib /app/BuhoCMS
+      - chmod +x /app/BuhoCMS/buhocms
+      - mkdir -p /app/bin
+      - ln -s /app/BuhoCMS/buhocms /app/bin/buhocms
+      - install -D installers/Flatpak/buhocms.svg -t /app/share/icons/hicolor/scalable/apps
+      - install -D installers/Flatpak/org.buhocms.BuhoCMS.desktop -t /app/share/applications
+      - install -D installers/Flatpak/org.buhocms.BuhoCMS.metainfo.xml -t /app/share/metainfo
+    sources:
+      - type: git
+        url: https://github.com/iakmds/buhocms.git
+        tag: v0.3.0
+        commit: 84d337921577a8cdca8dae17ced6e2967aa4596f
+      - type: archive
+        url: https://github.com/iakmds/buhocms/releases/download/v0.3.0/BuhoCMS-Linux-Portable.tar.gz
+        sha256: 3159db5103fc5b8056ab277dd3d56c1d9e50cb0b7d066686f75887e8a2664903
+        strip-components: 0
+  - name: hugo
```

Comment:
this will try to build hugo with autotools. It's unlilely to work.

*hint* even a binary distribution needs to be copied / installed.

Reply:
But how? In the actual program's case, it is copied to the app/bin folder, but where do these modules go to?

Reply:
it still is not doing it.

Reply:
Yes, but where do these modules go to, so that BuhoCMS will be able to run them?